packages/mobile/capacitor/plugins/secure-filesystem-access/android/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-all.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```
We should use the latest patch release from the 7.0.x line